lightwood/mixers/nn.py

```diff
@@ -447,6 +447,10 @@ def _predict(self, when_data_source, include_extra_data=False):
             if include_extra_data:
                 predictions[output_column]['encoded_predictions'] = output_trasnformed_vectors[output_column]
 
+            if 'predict_proba' in decoded_predictions:
+                predictions[output_column]['class_distribution'] = decoded_predictions['predict_proba']
+                predictions[output_column]['class_labels'] = decoded_predictions['class_labels']
```

Comment:
Why not keep consistent naming ? i.e call `class_distribution` here `predict_proba` or vice versa, replace `predict_proba` key with `class_distribution`